module_modo.lua

```diff
@@ -268,7 +269,7 @@ function Module:HandleEmojiAdd(userId, message)
 				},
 				{
 					name = "Message content",
-					value = content or "<empty>"
+					value = (#content > 0 and content) or "empty"
```

Comment:
`<empty>` est devenu `empty`, ce qui ne va pas

Reply:
This will trigger an error with a nil content, as you check its size before checking if it's valid.